File: LondonDataServices.IDecide.Portal.Server/Models/PatientSearches/PatientLookup.cs
// ---------------------------------------------------------
// Copyright (c) North East London ICB. All rights reserved.
// ---------------------------------------------------------

using System.Collections.Generic;

namespace LondonDataServices.IDecide.Portal.Server.Models.PatientSearches
{
    public class PatientLookup
    {
        public SearchCriteria SearchCriteria { get; set; }
        public List<Patient> Patients { get; set; }
    }
}

File: LondonDataServices.IDecide.Manage.Client/src/models/patients/patientLookup.ts
import { SearchCriteria } from "../searchCriterias/searchCriteria";
import { Patient } from "./patient";

export class PatientLookup {
    public searchCriteria: SearchCriteria;
    public patients: Patient[];

    constructor(searchCriteria?: Partial<SearchCriteria>, patients?: Patient[]) {
        this.searchCriteria = new SearchCriteria(searchCriteria ?? {});
        this.patients = patients ?? [];
    }
}